pattern-library/aquent.html

```diff
@@ -44,7 +44,120 @@
       <div id="content" class="content">
         <div class="row">
           <div class="columns medium-10">
-            <section id="feature-1">
+            <section id="colors">
+              <h3>Colors</h3>
+              <p>To do: these color require additional documentation.</p>
+              <h4>Foundation Colors:</h4>
+              <div class="row small-up-5 margin-bottom-2x">
+                <div class="column margin-bottom-1x">
+                  <div class="swatch bg-primary">primary: #4cc1e0</div>
+                </div>
+                <div class="column margin-bottom-1x">
+                  <div class="swatch bg-secondary">secondary: #f8971d</div>
+                </div>
+                <div class="column margin-bottom-1x">
+                  <div class="swatch bg-yellow" style="background: #fed301;">yellow: #fed301</div>
```

Comment:
I don't think you need to add a style attribute here. The `bg-yellow` should just show up, I think. Same applies below.